packages/headless-react/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@coveo/headless-react",
+  "version": "0.0.0",
+  "description": "React utilities for SSR (Server Side Rendering) with headless",
+  "homepage": "https://docs.coveo.com/en/headless/latest/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/coveo/ui-kit.git",
+    "directory": "packages/headless-react"
+  },
+  "keywords": [
+    "headless",
+    "react",
+    "nextjs",
+    "ssr"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc --project ./tsconfig.build.json",
+    "test": "jest",
+    "test:watch": "jest --watch --colors --no-cache",
+    "lint": "eslint ."
+  },
+  "dependencies": {
```

Comment:
I assume there was discussion about this that I might have missed, but should this be a peer dependency instead 🤔  ?

My first intuition would be that implementers would already install/need `@coveo/headless` no matter what, and that they can optionally install this as an... "addon" kinda thing ?